### ISSUE TITLE
LabVIEW 2019 SP1 crashes on launch after install of 1.0.0.62

### ISSUE THREAD

**Issue by pollockm** [JKISoftware/Caraya#59]
After installing the 1.0.0.62 release, LabVIEW 2019 SP1 now crashes on every launch.  This occurred on multiple machines.  Other LV versions on the same machine (with no Caraya installed) do not crash on launch, 2019 SP1 was working fine prior to install.

This was the first installation of Caraya onto these machines, so a clean install, not an upgrade, scenario.

Not sure if this affects any other versions.  Has this impacted anyone else?

![image](https://user-images.githubusercontent.com/5288961/71683982-a8780a80-2d59-11ea-8126-3e201358d219.png)

**Comment by pollockm** [comment]
If I install the dependencies manually before attempting to install Caraya, LabVIEW is happy.

**Comment by francois-normandin** [comment]
I don't know if this is perhaps related to one of the files accessible from the Tools menu, which gets loaded right away. I'll double-check the dependencies list in the vipb file. I have seen this behavior once in the past, but recompiling the vip file solved the issue. It was not reproducible back then, so I'll give this a shot.

**Comment by francois-normandin** [comment]
I've double-checked the dependencies list and all were included. 
I did not try a clean-install for any versions as I don't have a clean VM, so if anyone can give this a shot and report whether LabVIEW crashes on a clean install, it would be greatly appreciated.

**Comment by francois-normandin** [comment]
I will close this issue. I have not been able to reproduce with LabVIEW 2013, 2014, 2015, 2017 and 2019 SP1.
Since the problem might have been either local machine or a badly compiled previous package, this should be resolved. Please reopen this case if the problem still exists or re-appears.

**Comment by dnattinger** [comment]
I ran into this issue today in LabVIEW 2020. Upgrading to 2020 SP1 did not fix the crash on launch. For anyone reading this case who runs into the same issue, I was able to fix the crash on launch with the following manual procedure:

1. Rename _[LabVIEW 20xx]\project_ to _[LabVIEW 20xx]\_project_.
2. Launch LabVIEW. It shouldn't crash.
3. While LabVIEW is open, go into the file system and rename the afore-mentioned folder back to _project_.
4. Open _[LabVIEW 20xx]\project\masscompile.llb\mass compile.vi_. It will run when opened.
5. Point the Mass Compile to _[LabVIEW 20xx]\project_. 
6. After the mass compile is complete, restart LabVIEW. It should restart without crashing.
